dds/DCPS/transport/rtps_udp/RtpsUdpDataLink.cpp

```diff
@@ -2422,9 +2421,9 @@ RtpsUdpDataLink::RtpsWriter::gather_gaps_i(const RepoId& reader,
     bitmap.length(bitmap_num_longs(base, gaps.high()));
     (void) gaps.to_bitmap(bitmap.get_buffer(), bitmap.length(), num_bits);
   } else {
-    bitmap.length(1);
+    bitmap.length(0);
```

Comment:
Same issue with this one.

Reply:
Also, I am going to remove the  num_bits = 0 since it is initialized to 0 when it is declared.